checkpoint/side_milestone_handler_test.go

```diff
@@ -33,27 +33,6 @@ func (suite *SideHandlerTestSuite) TestSideHandleMsgMilestone() {
 
 	borChainId := "1234"
 
-	suite.Run("Failure-Before Hard Fork", func() {
```

Comment:
why removed this?

Reply:
This was related to HF check in sidehandler.